added support for 2.12.nightly and 2.13.nightly and some non stable versions (#736)
VirtusLab/scala-cli: website/docs/commands/compile.md

The nightly builds of Scala compiler are unstable ones which are published on a nightly basis.

> For using the latest Scala 2 and Scala 3 nightly builds, you should pass `2.nightly` and `3.nightly`, respectively. You can also request the last `2.12.nightly` and `2.13.nightly` versions. Currently, the`2.13.nightly` version is the same as `2.nightly`

Reviewer: ```suggestion
To use the latest Scala 2 and Scala 3 nightly builds, pass `2.nightly` and `3.nightly`, respectively. You can also request the last `2.12.nightly` and `2.13.nightly` versions. `2.13.nightly` is the same as `2.nightly`.
```
